tests/garage/tf/algos/test_npo.py

```diff
@@ -17,34 +18,34 @@
 class TestNPO(TfGraphTestCase):
     def test_npo_pendulum(self):
         """Test NPO with Pendulum environment."""
-        logger.reset()
-        env = TfEnv(normalize(gym.make("InvertedDoublePendulum-v2")))
-        policy = GaussianMLPPolicy(
-            env_spec=env.spec,
-            hidden_sizes=(64, 64),
-            hidden_nonlinearity=tf.nn.tanh,
-            output_nonlinearity=None,
-        )
-        baseline = GaussianMLPBaseline(
-            env_spec=env.spec,
-            regressor_args=dict(hidden_sizes=(32, 32)),
-        )
-        algo = NPO(
-            env=env,
-            policy=policy,
-            baseline=baseline,
-            batch_size=2048,
-            max_path_length=100,
-            n_itr=10,
-            discount=0.99,
-            gae_lambda=0.98,
-            policy_ent_coeff=0.0,
-            plot=False,
-        )
-        last_avg_ret = algo.train(sess=self.sess)
-        assert last_avg_ret > 20
+        with LocalRunner() as runner:
+            logger.reset()
+            env = TfEnv(normalize(gym.make("InvertedDoublePendulum-v2")))
+            policy = GaussianMLPPolicy(
+                env_spec=env.spec,
+                hidden_sizes=(64, 64),
+                hidden_nonlinearity=tf.nn.tanh,
+                output_nonlinearity=None,
+            )
+            baseline = GaussianMLPBaseline(
+                env_spec=env.spec,
+                regressor_args=dict(hidden_sizes=(32, 32)),
+            )
+            algo = NPO(
+                env=env,
+                policy=policy,
+                baseline=baseline,
+                max_path_length=100,
+                discount=0.99,
+                gae_lambda=0.98,
+                policy_ent_coeff=0.0,
+                plot=False,
```

Comment:
is `plot=` really needed in the algorithm anymore? why not have the sampler plot? the plotter is basically a sampler thread with a window attached

Reply:
No, it's not. It's a parameter of `Runner.train()` now. I'll remove it. And I think the plotter doesn't have a sampler, but it saves the current policy and obtains a single trajectory using rollout() helper function.

Reply:
Sounds good.

Reply:
please resolve once you do this